### DISCUSSION TITLE
Question regarding Win10 setup

### DISCUSSION THREAD

**Discussion by BostonShaun** [awawa-dev/HyperHDR#196] [Q&A]
I've search a ton but find the majority of people using the RPi method.  Relatively new to all this and have spent hours trying to figure it out myself but I'm stuck.

Basically looking to find out if it is possible to prevent the HDMI/usb capture card connected to the PC from downgrading the resolution on the XBox series x.  Gaming is the only thing I want HyperHDR to run for.

Setup is XB > HDMI split to capture and monitor > capture to PC.

XBox stays 1440p/120hz through splitter, but as soon as the capture card is connected to the PC, it downgrades to 1080/60hz.

**Comment by awawa-dev** [comment]
Hi
System on which HyperHDR is running has nothing to do with the problem. Probably it is caused by your HDMI splitter when it detects USB grabber connected to its second port and I suspect it can't support 1440p (probably only Ezcap320 can do it for certain color spaces). So HDMI splitter reports it back to xbox and it selects resolution that it supported by HDMI splitter and the splitter reports mode supported by both devices. Some HDMI splitters can report modes of the priority HDMI output regardless whether or not the second HDMI output can support it. And sometimes it can even downgrade 1440p/4k to 1080p for that secondary devices if it can't support such resolution, without affecting the primary HDMI output. Please refer to your HDMI splitter manual if it can prioritize primary HDMI output and if it has a built in scaler for secondary HDMI output.

**Comment by tavalin** [comment]
You'll need a splitter that is capable of 4k 60hz. [This link](https://www.intel.com/content/www/us/en/support/articles/000023004/graphics.html) states that 1440p at 120hz is 13.27Gbps, which is slightly less than the bandwidth required for 4k 60hz (which is 14.93Gbps)

I use [this Ezcoo splitter](https://www.amazon.co.uk/dp/B083JVPXDY/ref=cm_sw_r_apan_glt_i_ZVRT5Z5FEHTQEQHV8M82?_encoding=UTF8&psc=1)  to do 4k 60hz so it should work for you.

You'll also need to be aware of the capabilities of your capture device but this Ezcoo splitter should allow you to downscale to a resolution supported by your capture device.

**Reply by awawa-dev** [reply]
BTW I've got this grabber and it works fine for me but one thing from the manual bothers me: "Ezcoo sp12h2: Max bandwidth: 600MHz Max baud rate: 6Gbps" ???

**Reply by BostonShaun** [reply]
Apprecite the responses guys.  I ended up getting that EZCOO splitter, but have tried every single different setup and still cannot get the XBox to stay on 1400p@120hz.  Wants either 1440p@60hz or 1080@120hz.

Any ideas?  I've updated the firmware as well.

**Reply by awawa-dev** [reply]
Well, you were lucky to update the firmware because it's not that simply due to missing libraries and usually we have to find them somewhere by ourselves ;) Anyway I suspect the procedure is like that:
- connect TV to HDMI OUT 1 , grabber to HDMI OUT 2
- set EDID switch to COPY1: it will forward modes supported by the TV and ignore if the grabber supports them or not
- enable scaller switch for HDMI 2, disable scaller switch for HDMI 1. It will downscale video (if not DolbyVision) for the grabber that is connected to HDMI OUT 2 port, but it leaves the signal intact for TV that is connected to HDMI OUT 1
